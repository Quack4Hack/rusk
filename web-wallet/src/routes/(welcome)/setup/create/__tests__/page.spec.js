import {
	afterEach,
	describe,
	expect,
	it
} from "vitest";
import { cleanup, render } from "@testing-library/svelte";
import Create from "../+page.svelte";

describe("Create", () => {
	afterEach(cleanup);

	it("should render the Terms of Service step of the Create flow", () => {
		const { container } = render(Create);

		expect(container.firstChild).toMatchSnapshot();
	});
});
