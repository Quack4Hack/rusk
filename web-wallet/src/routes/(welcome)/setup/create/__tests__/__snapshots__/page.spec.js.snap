// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Create > should render the Terms of Service step of the Create flow 1`] = `
<div>
  <div
    class="onboarding-wrapper s-p4ikGGAJA5YG"
  >
    <section
      class="flex flex-col flex-1"
    >
      <div
        class="dusk-card"
      >
        <header
          class="dusk-card__header"
        >
           
          <h3
            class="h4"
          >
            Terms & Privacy Policy
          </h3>
           
        </header>
         
        <div
          class="flex flex-col gap-1"
        >
          <p>
            Our 
            <a
              class="dusk-anchor"
              href="https://dusk.network/privacy-policy/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Terms & Privacy Policy
            </a>
            <!--&lt;Anchor&gt;-->
             govern your use of our services,
			including data handling and user responsibilities.
			Your privacy and security are our top priorities.
          </p>
           
          <div
            class="flex gap-1 w-100"
          >
            <a
              aria-disabled="false"
              class="dusk-anchor dusk-anchor-button dusk-anchor-button--variant--tertiary dusk-anchor-button--size--normal flex-1"
              href="/setup"
            >
              <span
                class="dusk-anchor-button__text"
              >
                Cancel
              </span>
              
            </a>
            <!--&lt;Anchor&gt;-->
            <!--&lt;AnchorButton&gt;-->
             
            <button
              class="dusk-button dusk-button--type--button dusk-button--variant--secondary dusk-button--size--normal flex-1"
              type="button"
            >
              <span
                class="dusk-button__text"
              >
                Accept
              </span>
            </button>
            <!--&lt;Button&gt;-->
          </div>
        </div>
      </div>
      
      <!--&lt;Card&gt;-->
    </section>
    <!--&lt;TermsOfService&gt;-->
  </div>
  
  <!--&lt;+page&gt;-->
</div>
`;
